File: Tugas2/python/proc.py
class proc():
    __name = ""
    __price = 0

    # construct
    def __init__(self, name = "", price = 0):
        self.__type = type
        self.__name = name
        self.__price = price

    # set dan get
    def setname(self, name):
        self.__name = name
    
    def getname(self):
        return self.__name

    def setprice(self, price):
        self.__price = price
    
    def getprice(self):
        return self.__price

    # output
    def printproc(self):
        print("name : " + str(self.__name))
        print("price : " + str(self.__price))

File: Tugas1/proc.cpp
#include <iostream>
#include <string>

using namespace std;

class proc{
	private:
		string name;
        int pricep;
	
	public:
		//Construct
		proc(){}
		
		proc(string name, int pricep){
			this->name = name;
			this->pricep = pricep;
		}
		
		//Set dan Get
		void setname(string name)
        {
            this->name = name;
        }
		string getname(){
			return this->name;
		}
		
		void setpricep(int pricep){
			this->pricep = pricep;
		}
		int getpricep(){
			return this->pricep;
		}

        void printprosesor(){
            cout << "Nama  Procesor : " << getname() << endl;
            cout << "Harga Procesor : " << getpricep() << endl;
        }	
		
		~proc(){
		}
};